frontend/src/components/Main/OptionsForm/OptionsForm.js

```diff
@@ -102,6 +106,76 @@ const OptionsForm = props => {
                     </select>
                 </td>
             ) : null}
+            {props.modelType === mc.MODEL_NESTED_DICHOTOMOUS ? (
+                <>
+                    <td>
+                        <select
```

Comment:
use the `components/common/SelectInput` here and elsewhere in this file?

Reply:
using SelectInput tag can be a new PR i think since onChange function with parameters and for select options on our options form we have 3 inputs.

Reply:
sounds good to me; i was actually thinking the same, since it may be broader than this